tests/integration/reconcile/connectors/test_read_schema.py

```diff
@@ -80,32 +80,41 @@ def _get_snowflake_options(self):
         return opts
 
 
-@pytest.mark.skip(reason="Run in acceptance environment only")
-def test_sql_server_read_schema_happy(mock_spark):
+def test_sql_server_read_schema_happy(mock_spark: SparkSession) -> None:
     mock_ws = create_autospec(WorkspaceClient)
     connector = TSQLServerDataSourceUnderTest(mock_spark, mock_ws)
 
     columns = connector.get_schema("labs_azure_sandbox_remorph", "dbo", "reconcile_in")
     assert columns
 
 
-def test_databricks_read_schema_happy(mock_spark):
+def test_databricks_read_schema_happy(mock_spark: SparkSession) -> None:
```

Comment:
Hmm, it's not a change that's been introduced by this PR, but I'm a bit confused about the `mock_spark` fixture: it doesn't seem to be a mock at all but rather a real spark session.

What is this connected to? Is it connected to the acceptance Databricks workspace?